src/services/iamRole/schema.graphql

```diff
@@ -5,6 +5,7 @@ type awsIamRole implements awsBaseService @key(fields: "id") {
   assumeRolePolicy: awsIamJSONPolicy
   description: String @search(by: [hash, regexp])
   createdAt: String @search(by: [hash, regexp])
+  lastUsedDate: String @search(by: [hash, regexp])
```

Comment:
@M1kep thanks for contributing! As this is an ISO string field, it can actually be defined in the schema as a `DateTime` field. See the Dgraph docs here for more info: https://dgraph.io/docs/graphql/schema/search/#datetime

Reply:
After messing around with the DateTime instead of String, it still seems like String may be a better fit. This is mainly due to RoleLastUsed not always being present(In cases such as when the last used date is more than 400 days ago)

Reply:
Not too sure what error I was encountering before, but going to chalk it up to a misconfiguration on my part. I've pushed some changes that seem to support the DateTime field properly now

Reply:
Sounds good! Ill give this a test tomorrow. Thanks again for the contribution! @m-pizarro Can you also give this a test?

Reply:
Great, thanks for the contribution! @tyler-dunkel yes of course, I'll do some tests as well.